docs/4.3/aws-oss-guide.md

```diff
@@ -382,15 +273,13 @@ To upgrade to a newer version of Teleport:
 - Back up `/etc/teleport.yaml`, `/etc/teleport.d/` and the contents of `/var/lib/teleport`
 - Launch a new instance with the correct AMI for a newer version of Teleport
 - Copy `/etc/teleport.yaml`, `/etc/teleport.d/` and `/var/lib/teleport` to the new instance, overwriting anything that already exists
-- Copy  and its contents should also be backed up and copied over to the new instance.
+- Copy and its contents should also be backed up and copied over to the new instance.
```

Comment:
This sentence seems weird here. What are we trying to say. :)

Reply:
ha, fix a space.. fix the line.